Apps/Playground/0.65/ios/Playground.xcodeproj/project.pbxproj

```diff
@@ -485,6 +484,7 @@
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
 				CURRENT_PROJECT_VERSION = 1;
+				DEVELOPMENT_TEAM = UBF8T346G9;
```

Comment:
I don't think you want to check in these `DEVELOPMENT_TEAM` changes.

Reply:
Well, that's annoying. We have to undo the dev team before merging? I've never had to do that. I wonder what's different?

Reply:
Weird, I don't know what happened, but it didn't add them this time. I was having trouble with setting the dev team, so maybe I touched something that caused xcode to write it out. Anyways, it's removed now.